index.js

```diff
@@ -14,8 +14,10 @@
 module.exports = {
   parser: 'babel-eslint',
   extends: 'airbnb',
+  plugins: ["backpack"],
   rules: {
     'valid-jsdoc': ['error'],
+    "backpack/use-tokens": 2,
```

Comment:
Use single quotes like the rest of the file e.g:
```
plugins: ['backpack'],
```
and
```
'backpack/use-tokens': 'error',
```

Note i've also changed `2` to `'error'` to be consistent with the line above.